src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -83,60 +83,65 @@ ov::runtime::Tensor generate(const ov::element::Type& elemType,
 }
 } // namespace Activation
 
-ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Abs>& node,
-                             size_t port,
-                             const ov::element::Type& elemType,
-                             const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape);
-}
-
-ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Acos>& node,
-                             size_t port,
-                             const ov::element::Type& elemType,
-                             const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape, InputGenerateData(-1, 2, 32768, 1));
-}
-
-ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Asin>& node,
-                             size_t port,
-                             const ov::element::Type& elemType,
-                             const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape, InputGenerateData(-1, 2, 32768, 1));
-}
-
-ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Atan>& node,
-                             size_t port,
-                             const ov::element::Type& elemType,
-                             const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape, InputGenerateData(-1, 2, 32768, 1));
-}
-
-ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Ceiling>& node,
+ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::HardSigmoid>& node,
                              size_t port,
                              const ov::element::Type& elemType,
                              const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape, InputGenerateData(-1000, 2000, 32768, 1));
-}
+    switch (port) {
+        case 1: {
+            std::vector<float> alpha(node->get_input_shape(1).size(), 0.2f);
+            return ov::test::utils::create_tensor<float>(elemType, targetShape, alpha, alpha.size());
```

Comment:
Hm. Lets generate tensor via `fill_tensor` method? The method is able to fill tensor by same values

Reply:
I cannot find this method. Could you please show what header contains it?

Reply:
done